### PR TITLE
Fix upgrade script

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>
https://github.com/stolostron/backlog/issues/18728
**Description of the change:** Fixes upgrade script

**Motivation for the change:**

### CODE REVIEW

File: upgrade.sh
Comment (Kyl-Bempah) [inline upgrade.sh:26]: Is there a reason why we didn't just update the TARGET_NAMESPACE variable or create a new one for this part?

File: upgrade.sh
Comment (Kyl-Bempah) [inline upgrade.sh:55]: Does this change functionality for pre-2.5 upgrade steps, we still need this to work for all supported z streams, including those before MCE

File: upgrade.sh
Comment (Kyl-Bempah) [inline upgrade.sh:34]: Has this been tested on the other z streams?

**Review by Kyl-Bempah — COMMENTED** [review]
I've left comments regarding some questions & things to be aware before we can merge, currently can't verify due to the rename

File: upgrade.sh
Comment (zkayyali812) [reply upgrade.sh:26]: TARGET_NAMESPACE should stay the same. The subscription/CSV/installplan will all be stood up in the TARGET_NAMESPACE, whereas only the catalogsource is stood up in `openshift-marketplace`

File: upgrade.sh
Comment (zkayyali812) [reply upgrade.sh:55]: This does not change functionality for pre-2.5 upgrade steps. This should still work as expected for all supported z-streams

File: upgrade.sh
Comment (zkayyali812) [reply upgrade.sh:34]: I have explicitly only test a 2.2.9 to a 2.3.4 update using this code.

File: upgrade.sh
Comment (Kyl-Bempah) [reply upgrade.sh:26]: Do we want to make that another variable or leave it explicitly defined?

**Comment by Kyl-Bempah** [comment]
So general consensus of CICD is to hold of on any further non-critical merges until we get deploys back up and running, post rename

**Comment by zkayyali812** [comment]
@Kyl-Bempah Do you think this can be merged soon, or are PRs still on hold?

**Comment by Kyl-Bempah** [comment]
Testing to get 2.5 canaries off the ground first then I'll switch to testing upgrade with this PR

**Review by Kyl-Bempah — APPROVED** [review]
LGTM

**Pull request merged**
